streamly.cabal

```diff
@@ -366,7 +373,10 @@ library
                      , Streamly.Data.Unfold
                      , Streamly.Data.Unicode.Stream
 
-                    -- IO devices
+                     -- Memory storage
+                     ,  Streamly.Internal.Memory.Malloc
```

Comment:
Should we call the `Streamly.Internal.Foreign.Malloc`, it exports only Foreign ptr malloc. The `Memory` hierarchy anyway goes away with the Array module renaming. Also this would be consistent with the Foreign module in base.